ports/flatcc/portfile.cmake

```
@@ -0,0 +1,34 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO dvidelabs/flatcc
+    REF "v${VERSION}"
+    SHA512 46ba5ca75facc7d3360dba797d24ae7bfe539a854a48831e1c7b96528cf9594d8bea22b267678fd7c6d742b6636d9e52930987119b4c6b2e38d4abe89b990cae
+    HEAD_REF master
+    PATCHES
+        fix_install_dir.patch
+)
+string(COMPARE EQUAL ${VCPKG_LIBRARY_LINKAGE} "dynamic" FLATCC_DYNAMIC)
+
+if(VCPKG_TARGET_IS_WINDOWS)
+    list(APPEND EXTRA_OPTIONS "-DFLATCC_INSTALL_LIB=bin")
```

Should this go in bin? Why not include? It looks like this installs headers.

---

I don't see how this is related to headers, this controls the library install path

https://github.com/dvidelabs/flatcc/blob/be12a1f74bbbda9bbad749f5fbf56c342c2878cb/CMakeLists.txt#L382-L387

should probably be

```suggestion
    list(APPEND EXTRA_OPTIONS "-DFLATCC_INSTALL_LIB=lib")
```

---

oops, I was confused and didn't see the _LIB suffix at the end. But yes, is should be set to "lib"

---

this actually is the default, removed.